.circleci/config.yml

```diff
@@ -889,11 +889,15 @@ workflows:
             - playwright-build
 
   build-and-deploy-all-apps-workflow:
-    # check all studies, launch GAE deploy if a study has changed.
+    # check all studies, launch GAE deploy only if a study has changed.
     when:
       and:
         - not: << pipeline.parameters.do-builds >>
         - not: << pipeline.parameters.scheduled-playwright >>
+        - not:
+            matches:
+              pattern: ".*-tests$"
+              value: << pipeline.parameters.api_call >>
```

Comment:
Specified pattern should exclude api calls `run-tests` and `run-e2e-tests` from this workflow.